tensorrt_llm/_torch/models/modeling_nanov2vlm.py

```
@@ -262,8 +262,9 @@ def __init__(self,
                  model_path: str,
                  config: transformers.PretrainedConfig,
                  tokenizer: transformers.AutoTokenizer,
-                 trust_remote_code: bool = True):
-        super().__init__()
+                 trust_remote_code: bool = True,
+                 **kwargs):
+        super().__init__(**kwargs)
         if not trust_remote_code:
             raise ValueError("trust_remote_code must be True for Phi4MM")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)